src/terminal/emoji.rs

```diff
@@ -22,6 +22,7 @@ pub static EYES: Emoji = Emoji("👀 ", "");
 pub static FILES: Emoji = Emoji("🗂️ ", "");
 pub static INBOX: Emoji = Emoji("📥 ", "");
 pub static INFO: Emoji = Emoji("💁 ", "");
+pub static LIST: Emoji = Emoji("📋 ", "");
```

Comment:
is this used anywhere?

Reply:
i cannot figure out why i added this. i think i was thinking that the interface here should maybe be wrangler account list? i can remove it for now... but also should it be wrangler account list?